tools/ci_build/github/azure-pipelines/orttraining-linux-ci-pipeline.yml

```diff
@@ -25,6 +25,17 @@ jobs:
       DockerBuildArgs: "--build-arg BUILD_UID=$( id -u )"
       Repository: onnxruntimecpubuild
 
+  - task: CmdLine@2
+    displayName: 'Get the latest ONNX for integration check'
+    condition: eq(variables['ONNX_INTEGRATION_WEEKLY_BUILD'], '1')
+    inputs:
+      script: |
+        cd $(Build.SourcesDirectory)/cmake/external/onnx
```

Comment:
I would suggest using cmake fetch content instead: https://cliutils.gitlab.io/modern-cmake/chapters/projects/fetch.html

Then we can add a cmake option and use it to control which onnx version to use: the master or the latest release.

Reply:
I will make the change. I have a draft in https://github.com/Microsoft/onnxruntime/tree/snnn/cmake_external branch.